presto-vertica/src/test/java/io/prestosql/plugin/vertica/TestVerticaIntegrationSmokeTest.java

```diff
@@ -0,0 +1,143 @@
+    @Override
+    public void testShowCreateTable()
+    {
+        // Replaced shippriority column's 'integer' with 'bigint'
+        assertThat((String) computeActual("SHOW CREATE TABLE orders").getOnlyValue())
+                // If the connector reports additional column properties, the expected value needs to be adjusted in the test subclass
+                .matches("CREATE TABLE \\w+\\.\\w+\\.orders \\Q(\n" +
+                        "   orderkey bigint,\n" +
+                        "   custkey bigint,\n" +
+                        "   orderstatus varchar(1),\n" +
+                        "   totalprice double,\n" +
+                        "   orderdate date,\n" +
+                        "   orderpriority varchar(15),\n" +
+                        "   clerk varchar(15),\n" +
+                        "   shippriority bigint,\n" +
+                        "   comment varchar(79)\n" +
+                        ")");
+    }
+
+    @Test
+    public void testSupportedTypes()
+    {
+        typeMappingTest("BOOLEAN", "true", "true");
+        typeMappingTest("TINYINT", "-128", "-128");
+        typeMappingTest("SMALLINT", "32767", "32767");
+        typeMappingTest("INT", "-2147483648", "-2147483648");
+        typeMappingTest("INT8", "2147483647", "2147483647");
+        typeMappingTest("INTEGER", "-2147483648", "-2147483648");
+        typeMappingTest("BIGINT", "-9223372036854775807", "-9223372036854775807");
+        typeMappingTest("DOUBLE PRECISION", "1.23", "1.23");
+        typeMappingTest("DOUBLE PRECISION", "'Infinity'", "CAST('Infinity' AS DOUBLE)");
+        typeMappingTest("DOUBLE PRECISION", "'NaN'", "CAST('NaN' AS DOUBLE)");
+        typeMappingTest("FLOAT", "4.56", "4.56");
+        typeMappingTest("FLOAT(1)", "1", "1");
+        typeMappingTest("FLOAT(53)", "11111111112222222222333333333344444444445555555555000", "CAST('11111111112222222222333333333344444444445555555555000' AS DOUBLE)");
+        typeMappingTest("FLOAT8", "88888888", "88888888");
+        typeMappingTest("NUMERIC", "1111111111222222222233.444444444455555", "1111111111222222222233.444444444455555"); // numeric(37, 15)
+        typeMappingTest("NUMERIC(10, 5)", "12345.67890", "12345.67890");
+        typeMappingTest("NUMERIC(38, 0)", "12345.67890", "12345.67890");
+        typeMappingTest("REAL", "12345.67890", "12345.67890");
+        typeMappingTest("CHAR", "'a'", "'a'"); // char(1)
+        typeMappingTest("CHAR(1)", "'b'", "'b'");
+        typeMappingTest("CHAR(65000)", "'my_char'", "'my_char'");
+        typeMappingTest("VARCHAR", "'my_varchar'", "'my_varchar'"); // varhcar(80)
+        typeMappingTest("VARCHAR(1)", "'a'", "'a'");
+        typeMappingTest("VARCHAR(65000)", "'my_varchar'", "'my_varchar'");
+        typeMappingTest("DATE", "'1970-01-01'", "DATE '1970-01-01'");
+        typeMappingTest("DATE", "'9999-12-31'", "DATE '9999-12-31'");
+    }
+
+    private void typeMappingTest(String verticaType, String verticaLiteral, String expectedLiteral)
```

Comment:
This should be Type mapping test class. 
See `TestPostgreSqlTypeMapping` for a reasonable template